Etabs_Adapter/CRUD/Read/Load.cs

```diff
@@ -136,48 +145,187 @@ private List<ILoad> ReadBarLoad(List<Loadcase> loadcases)
 
             foreach (Loadcase bhLoadcase in loadcases)
```

Comment:
This was here before, but might as well fix, for all load extractions.

Currently we are looping through and extracting all loads, one time for each loadcase, and then looping through the loads filtering out the ones corresponding to the case we are after. This is quite heavy, so suggesting to instead:

- Pull all loads
- Load through the values
- Check if loadcase is in provided list, if not, continue

Should be able to refactor to something like this instead:

```
            //foreach (Loadcase bhLoadcase in loadcases)
            //{
            if (m_model.FrameObj.GetLoadDistributed("All", ref nameCount, ref names, ref loadcase, ref myType, ref CSys, ref dir, ref rd1, ref rd2, ref dist1, ref dist2, ref val1, ref val2, eItemType.Group) != 0)
            {
                //Raise warning
                return new List<ILoad>();
            }

            for (int i = 0; i < nameCount; i++)
            {
                Loadcase bhLoadcase = loadcases.FirstOrDefault(x => x.Name == loadcase[i]);

                if (bhLoadcase == null)
                    continue;

                //if (bhLoadcase.Name != loadcase[i])
                //    continue;
```

I kept the commented out code for reference, ofc to be deleted.